base/linalg/uniformscaling.jl

```diff
@@ -56,6 +56,10 @@ ishermitian(J::UniformScaling) = isreal(J.λ)
 (-)(B::BitArray{2}, J::UniformScaling)      = Array(B) - J
 (-)(J::UniformScaling, B::BitArray{2})      = J - Array(B)
 
+for t in (:Symmetric, :Hermitian), op in (:+, :-)
+    @eval ($op)(X::$t, J::UniformScaling) = ($t)(($op)(X.data, J), Symbol(X.uplo))
```

Comment:
should test that Hermitian plus a complex UniformScaling with nonzero imag component currently should error